packages/scss/src/commons/core.scss

```diff
@@ -34,40 +34,31 @@ $overflow: 'hidden', 'auto', 'visible', 'scroll';
 
 /* Tokens */
 
-@mixin spacing($boxModel, $boxDirection, $key, $value, $suffix: '!important', $token: true) {
-	@if $token {
-		.pr-u-#{transform.camelize($boxModel)}#{transform.capitalize($boxDirection)}#{transform.capitalize($key)} {
-			#{$boxModel}#{if($boxDirection == '', '', '-')}#{$boxDirection}: var(--pr-t-spacings-#{$key}) #{$suffix};
-		}
-	} @else {
-		.u-#{transform.camelize($boxModel)}#{transform.capitalize($boxDirection)}#{transform.capitalize($key)} {
-			#{$boxModel}#{if($boxDirection == '', '', '-')}#{$boxDirection}: var(--spacings-#{$key}) #{$suffix};
-		}
+@mixin spacing($boxModel, $boxDirection, $key, $value, $suffix: '!important') {
+	.pr-u-#{transform.camelize($boxModel)}#{transform.capitalize($boxDirection)}#{transform.capitalize($key)} {
+		#{$boxModel}#{if($boxDirection == '', '', '-')}#{$boxDirection}: var(--pr-t-spacings-#{$key}) #{$suffix};
 	}
 }
 
-@mixin spacings($token: true) {
-	$spacings: config.$spacingsDeprecated;
-	@if $token {
-		$spacings: config.$spacings;
-	}
+@mixin spacings() {
+	$spacings: config.$spacings;
```

Comment:
You don't need to go through an intermediate variable, you can directly use `config.$spacing`.